tags/tags-impl/impl/pom.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0"?>
```

Comment:
The version numbers and dependencies here should be defined in dependencyManagement in master and some may already be defined.